### DISCUSSION TITLE
FileUpload component upload csv file not working

### DISCUSSION THREAD

**Discussion by elainecyx** [filamentphp/filament#14160] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.108

### How can we help you?

Environment:
```
Laravel v11.21.0
Filament v3.2.108
```
See code as below:
```
Forms\Components\FileUpload::make('attachment')
                            ->storeFiles(false)
                            ->fetchFileInformation(false)
                            ->acceptedFileTypes(['text/csv'])
                            ->maxSize(500)
```
It's test ok in my local laptop, the file can saved in `storage/app/livewire-tmp/`, but deployed to sandbox environment, the file is stuck in uploading.And didn't have any errors in log.
![image](https://github.com/user-attachments/assets/63230ca3-f6ef-4e58-ab6e-a5881f224887)
Even though I change to upload the file to GCS, it is still stuck in uploading. But can use Export Action to export data to GCS.
How can I fixed this? Thanks in advance!!

**Comment by leandrocfe** [comment]
console errors?

**Reply by elainecyx** [reply]
Thanks for your reminder. I did find the error in console as below:
![image](https://github.com/user-attachments/assets/85031781-cd38-426d-b59c-51879e7c2f42)

**Reply by leandrocfe** [reply]
Mixed content: http/https. Check the app_url in the .env and try to force https in your app

https://dev.to/frikishaan/how-to-force-https-in-laravel-e1g

**Reply by elainecyx** [reply, marked as answer]
Thanks for your help!! Finally it  is working for me as below code:
Add force schema https when environment is production Add bellow code to app/Providers/AppServiceProvider.php at boot function
```
if($this->app->environment('production')) {
    \URL::forceScheme('https');
}
```
And after apply above code, 401 Unauthorization happens.
In App\Http\Middleware\TrustProxies make sure the protected proxies is set like below:
` protected $proxies = '*';`
Now is ok.
